### PR TITLE
Feature identification

### PR DESCRIPTION
**Pull request by SimonDmz**
new component
light graphical update
business rules for campaign configurations

### CODE REVIEW

**Review by ddecrulle — COMMENTED** [review]
Too big for me I reviewed somes

File: src/components/panel-body/UEpage/atomicInfoTile.js [hunk 1]
```diff
@@ -1,83 +1,57 @@
...
-    (arr, { label }) => [
-      ...arr,
-      <Typography key={label} className={classes.label}>
-        {label !== undefined ? label : ''}
-      </Typography>,
-    ],
-    []
-  );
-  const values = data.reduce((arr, { label, value, favorite }) => {
-    const valid = value !== undefined && value !== '';
```
[the remark this answers is not in the document]
Comment (SimonDmz) [reply src/components/panel-body/UEpage/atomicInfoTile.js:48]: red line => 👋

File: src/components/panel-body/UEpage/atomicInfoTile.js [hunk 2]
```diff
@@ -1,83 +1,57 @@
...
-    return [
-      ...arr,
-      <div className={classes.row}>
-        {favorite !== undefined && <MaterialIcons type={favorite ? 'starFull' : 'starOutlined'} />}
```
[the remark this answers is not in the document]
Comment (SimonDmz) [reply src/components/panel-body/UEpage/atomicInfoTile.js:52]: red line => 👋

File: src/components/panel-body/UEpage/atomicInfoTile.js [hunk 3]
```diff
@@ -1,83 +1,57 @@
...
-        <Typography key={label} className={valid ? '' : classes.invisible}>
-          {value !== undefined && value !== '' ? value : '-'}
```
[the remark this answers is not in the document]
Comment (SimonDmz) [reply src/components/panel-body/UEpage/atomicInfoTile.js:54]: red line => same as before 😉

File: src/components/common/niceComponents/LabelledBoolean.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+const LabelledBoolean = ({ labelText, value = undefined }) => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.row}>
+      <Typography color="textSecondary" className={classes.rightMargin}>
+        {labelText}
+      </Typography>
+      {value !== undefined && <MaterialIcons type={`${value ? 'checked' : 'cross'}`} />}
```
[the remark this answers is not in the document]
Comment (SimonDmz) [reply src/components/common/niceComponents/LabelledBoolean.js:22]: compliance with  Java **B**oolean : true / false / undefined

**Pull request merged**